content/200-concepts/100-components/01-prisma-schema/06-relations/300-many-to-many-relations.mdx

```diff
@@ -282,6 +282,7 @@ Implicit m-n relations:
 
 - Use a specific [convention for relation tables](#conventions-for-relation-tables-in-implicit-m-n-relations)
 - Do **not** require the `@relation` attribute unless you need to [disambiguate relations](/concepts/components/prisma-schema/relations#disambiguating-relations) (this is not specific to implicit n-m relations).
+- If they use an `@relation` attribute for disambiguation, you can still not use the `fields` and `references` arguments as these are implicit and fixed for implicit m-n relations. 
```

Comment:
Mention that `references` (and `fields`) are not allowed when you use `@relation` on implicit m:n

Reply:
I've attempted to reword this and also include the information in the comment below, probably worth another check.

Reply:
Left comments on the new version. Much better already, but not there yet.